hack/cmd_util.sh

```diff
@@ -400,30 +400,12 @@ function os::cmd::internal::run_until_exit_code() {
 		fi
 		return 0
 	else
-		local cause=$(os::cmd::internal::assemble_try_until_code_causes "${cmd_succeeded}")
-
-		os::text::print_red_bold "FAILURE after ${time_elapsed}s: ${description}: ${cause}"
+		os::text::print_red_bold "FAILURE after ${time_elapsed}s: ${description}: the command timed out"
```

Comment:
@liggitt 
The current output already states how long the command is tried before giving up, so I don't think we gain anything from printing it twice. Were you commenting on the wording of `the command timed out` or specifically that you wanted to add that other piece of information to the message?